src/main.js

```diff
@@ -64,7 +64,7 @@ export default class {
 
 	maskString (rawString) {
 		return rawString.replace(this.sensitiveFields, (match, p1) => {
-			return p1 ? `${p1}="${MASK_SEQUENCE}"` : MASK_SEQUENCE;
+			return p1 ? `${p1}="${MASK_SEQUENCE}"` : match;
```

Comment:
I know you did not write this but what does p1 mean?

Reply:
The nth string found by a parenthesized capture group - so if we had a regex like `(.*)=(.*)`, p1 would be the first, p2 would be the second:

`email=test` -> `p1` is `email`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter

Reply:
So the `pn` represents parenthesis `n` of the regex? (Trying to work out why 'p').